src/mpcCoreKit.ts

```diff
@@ -461,10 +461,10 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     try {
       const factorPub = getPubKeyPoint(factorKey);
       await this.copyOrCreateShare(shareType, factorPub);
-      await this.backupShareWithFactorKey(factorKey);
+      await this.backupMetadataShare(factorKey);
       await this.addFactorDescription(factorKey, shareDescription, additionalMetadata);
       if (!this.options.manualSync) await this.tKey.syncLocalMetadataTransitions();
-      return factorKey.toString("hex").padStart(64, "0");
+      return factorKey.toArrayLike(Buffer, "be", SCALAR_LEN).toString("hex");
```

Comment:
we should use ``.toString("hex", 64);```

Reply:
I agree this can be shortened using `factorKey.toString(...)`
However, we shouldn't use magic numbers like `64`. Instead we should use constants, or even better common classes or functions for serializing elliptic curve scalars and points.
Already added a class for `Point` for points before.
In this case, added a utility function `scalarBNToBufferSEC1` and using this here. It outputs a buffer of the correct length which can then be easily converted into different representations (e.g., "hex" or "base64"). This will hopefully help us avoid these discussions in the future.